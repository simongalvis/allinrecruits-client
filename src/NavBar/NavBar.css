.NavBar{
    background-color: rgb(75, 75, 75);
}
h1{
    margin-top: 0;
}
header{
    margin-bottom: 4%;
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 5%;
    background-color:white;
    padding: 3px 16px;
    border-bottom: 1px solid rgb(231, 231, 231);
    min-height: 55px;
    
}
#main-logo{
    padding-top: 7%;
    height:59px;
    width:170px;
    margin:0;
    padding:0;
}
