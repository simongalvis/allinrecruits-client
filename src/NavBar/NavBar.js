import React from 'react'
import './NavBar.css'
import logo from '../images/all-in-logo.png'
import { Link } from 'react-router-dom'

class NavBar extends React.Component{



    render(){
        return(
            <div className="NavBar">
                <header>
                    <Link to="/">
                        <img src={logo} alt="all-in-education-logo" id="main-logo"/>
                    </Link>
                    <a href="/admin-login">
                        <button  id="admin-login-button">Admin Login</button>
                    </a>
                </header>
                
                
            </div>
        )
    }
}

export default NavBar;