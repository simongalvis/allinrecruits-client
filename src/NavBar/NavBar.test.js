import React from 'react'
import ReactDOM from 'react-dom'
import NavBar from './NavBar'
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom'

describe(`NavBar component`, () =>{
    it(`renders without crashing`, () =>{
        const div = document.createElement('div');

        ReactDOM.render(<BrowserRouter><NavBar/></BrowserRouter>, div)

        ReactDOM.unmountComponentAtNode(div);
    })
    it(`Renders UI as expected`, () =>{
        const tree = renderer
            .create(<BrowserRouter><NavBar/></BrowserRouter>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})