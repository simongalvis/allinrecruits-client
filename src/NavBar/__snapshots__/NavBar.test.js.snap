// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component Renders UI as expected 1`] = `
<div
  className="NavBar"
>
  <header>
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="all-in-education-logo"
        id="main-logo"
        src="all-in-logo.png"
      />
    </a>
    <a
      href="/admin-login"
    >
      <button
        id="admin-login-button"
      >
        Admin Login
      </button>
    </a>
  </header>
</div>
`;
